### DISCUSSION TITLE
SPA unauthenticated

### DISCUSSION THREAD

**Discussion by coloralp** [dedoc/scramble#274] [Q&A]
I have a project as SPA project . I can see docs when write 'docs/api' but after when ı try to get data with token(I m using sanctum)  
ı faced Unauthenticated error .
I added this code to provider file
 Scramble::extendOpenApi(function (OpenApi $openApi) {
            $openApi->secure(
                SecurityScheme::http('bearer')
            );
        });

I checked  header when ı try to send request 

Token goes like in header part(Authorization: Bearer $bearerToken) 

In addition  when ı try to send same request with POSTMAN with same token header part same request is success ı can take the data with token

**Comment by romalytvynenko** [comment]
@coloralp should be fixed in https://github.com/dedoc/scramble/releases/tag/v0.8.5, please check it out. Feel free to re-open discussion if not fixed.
